Crash Report 3.3.1: NPE MainMenu.isInTabSwitcher

```
java.lang.RuntimeException: Unable to resume activity {org.kiwix.kiwixmobile/org.kiwix.kiwixmobile.main.KiwixMainActivity}: java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=1236, result=-1, data=Intent {  launchParam=MultiScreenLaunchParams { mDisplayId=0 mBaseDisplayId=0 mFlags=0 }(has extras) }} to activity {org.kiwix.kiwixmobile/org.kiwix.kiwixmobile.main.KiwixMainActivity}: java.lang.NullPointerException: Attempt to read from field 'boolean org.kiwix.kiwixmobile.core.main.MainMenu.isInTabSwitcher' on a null object reference
at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3873)
at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3914)
at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3070)
at android.app.ActivityThread.-wrap14(ActivityThread.java)
at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1659)
at android.os.Handler.dispatchMessage(Handler.java:102)
at android.os.Looper.loop(Looper.java:154)
at android.app.ActivityThread.main(ActivityThread.java:6816)
at java.lang.reflect.Method.invoke(Native Method)
at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1563)
at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1451)
Caused by: java.lang.RuntimeException: Failure delivering result ResultInfo{who=null, request=1236, result=-1, data=Intent {  launchParam=MultiScreenLaunchParams { mDisplayId=0 mBaseDisplayId=0 mFlags=0 }(has extras) }} to activity {org.kiwix.kiwixmobile/org.kiwix.kiwixmobile.main.KiwixMainActivity}: java.lang.NullPointerException: Attempt to read from field 'boolean org.kiwix.kiwixmobile.core.main.MainMenu.isInTabSwitcher' on a null object reference
at android.app.ActivityThread.deliverResults(ActivityThread.java:4556)
at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3847)
... 10 more
Caused by: java.lang.NullPointerException: Attempt to read from field 'boolean org.kiwix.kiwixmobile.core.main.MainMenu.isInTabSwitcher' on a null object reference
at org.kiwix.kiwixmobile.core.main.CoreMainActivity.onActivityResult(CoreMainActivity.java:3)
at android.app.Activity.dispatchActivityResult(Activity.java:7329)
at android.app.ActivityThread.deliverResults(ActivityThread.java:4552)
... 11 more
```

--- a/core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.java
+++ b/core/src/main/java/org/kiwix/kiwixmobile/core/main/CoreMainActivity.java
@@ -1450,7 +1450,7 @@ public abstract class CoreMainActivity extends BaseActivity
     switch (requestCode) {
       case MainMenuKt.REQUEST_FILE_SEARCH:
         if (resultCode == RESULT_OK) {
-          boolean wasFromTabSwitcher = mainMenu.isInTabSwitcher();
+          boolean wasFromTabSwitcher = mainMenu != null && mainMenu.isInTabSwitcher();
           hideTabSwitcher();
           String title =
             data.getStringExtra(TAG_FILE_SEARCHED).replace("<b>", "").replace("</b>", "");

